create_lmodrc.py

```diff
@@ -95,14 +95,13 @@
 end
 
 local function eessi_openmpi_load_hook(t)
-    -- disable smcuda BTL when loading OpenMPI module for aarch64/neoverse_v1,
+    -- disable smcuda BTL when loading OpenMPI module,
     -- to work around hang/crash due to bug in OpenMPI;
-    -- see https://gitlab.com/eessi/support/-/issues/41
+    -- Fix should be merged via https://github.com/EESSI/software-layer/pull/488
```

Comment:
I would keep the reference to the support ticket?
```suggestion
    -- see https://gitlab.com/eessi/support/-/issues/41
    -- Fix should be merged via https://github.com/EESSI/software-layer/pull/488